src/main/kotlin/team/mozu/dsm/application/port/out/item/CommandItemPort.kt

```diff
@@ -4,4 +4,6 @@ import team.mozu.dsm.domain.item.model.Item
 
 interface CommandItemPort {
     fun save (item: Item) : Item
+
+    fun delete(item : Item)
```

Comment:
```suggestion
    fun save (item: Item): Item

    fun delete(item: Item)
```

띄어쓰기를 안하고 이렇게 컨벤션을 맞추는게 좋을 것 같아요!

Reply:
2c3535a28721e4dd95e8dc9f179dd5bca9cb9211